conf/sage.config

```diff
@@ -0,0 +1,58 @@
+params {
+  config_profile_description = 'The Sage Bionetworks profile'
+  config_profile_contact = 'Bruno Grande (@BrunoGrandePhD)'
+  config_profile_url = 'https://github.com/Sage-Bionetworks-Workflows'
+}
+
+process {
+
+  cpus   = { check_max( 1    * slow(task.attempt), 'cpus'   ) }
+  memory = { check_max( 6.GB * task.attempt,       'memory' ) }
+  time   = { check_max( 24.h * task.attempt,       'time'   ) }
+
+  errorStrategy = { task.exitStatus in [143,137,104,134,139,247] ? 'retry' : 'finish' }
+  maxRetries    = 5
+  maxErrors     = '-1'
+
+  // Process-specific resource requirements
```

Comment:
I'm not sure that there's necessarily a positive correlation between memory requirements and expected runtime (for example tools/scripts that use only one cpu and only a small amount of RAM but run for a long time), but I don't have a problem with this since these are always customizable by the user